dev-requirements.txt

```diff
@@ -0,0 +1,2 @@
+jupyter
```

Comment:
I suggest to created conda env instead. It is more convenient and there you can use a Python 3. Just put the same into the `enviromnet.yml` file.

Reply:
done